include/rdma/fi_eq.h

```diff
@@ -74,8 +72,8 @@ struct fid_wait {
 };
 
 struct fi_mutex_cond {
-	pthread_mutex_t		*mutex;
-	pthread_cond_t		*cond;
+	struct pthread_mutex_t	*mutex;
+	struct pthread_cond_t	*cond;
```

Comment:
right, but in this case we have to add construct like 
```
#if Linux
use-pthread-types
#else
use-windows-threading
#endif
```
and same for includes of interface headers (pthread or windows.h). previous PR was declined due to too many if-defs :)